packages/flutter_tools/test/integration.shard/expression_evaluation_test.dart

```diff
@@ -43,52 +43,46 @@ void batch1() {
     );
   }
 
-  testWithoutContext('flutter run expression evaluation - can evaluate trivial expressions in top level function', () async {
-    await initProject();
+  testWithoutContext('can evaluate trivial expressions in top level function', () async {
     await flutter.run(withDebugger: true);
     await breakInTopLevelFunction(flutter);
     await evaluateTrivialExpressions(flutter);
-    await cleanProject();
   });
 
-  testWithoutContext('flutter run expression evaluation - can evaluate trivial expressions in build method', () async {
-    await initProject();
+  testWithoutContext('can evaluate trivial expressions in build method', () async {
     await flutter.run(withDebugger: true);
     await breakInBuildMethod(flutter);
     await evaluateTrivialExpressions(flutter);
-    await cleanProject();
   });
 
-  testWithoutContext('flutter run expression evaluation - can evaluate complex expressions in top level function', () async {
-    await initProject();
+  testWithoutContext('can evaluate complex expressions in top level function', () async {
     await flutter.run(withDebugger: true);
     await breakInTopLevelFunction(flutter);
     await evaluateComplexExpressions(flutter);
-    await cleanProject();
   });
 
-  testWithoutContext('flutter run expression evaluation - can evaluate complex expressions in build method', () async {
-    await initProject();
+  testWithoutContext('can evaluate complex expressions in build method', () async {
     await flutter.run(withDebugger: true);
     await breakInBuildMethod(flutter);
     await evaluateComplexExpressions(flutter);
-    await cleanProject();
   });
 
-  testWithoutContext('flutter run expression evaluation - can evaluate expressions returning complex objects in top level function', () async {
-    await initProject();
+  testWithoutContext('can evaluate expressions returning complex objects in top level function', () async {
     await flutter.run(withDebugger: true);
     await breakInTopLevelFunction(flutter);
     await evaluateComplexReturningExpressions(flutter);
-    await cleanProject();
   });
 
-  testWithoutContext('flutter run expression evaluation - can evaluate expressions returning complex objects in build method', () async {
-    await initProject();
+  testWithoutContext('can evaluate expressions returning complex objects in build method', () async {
     await flutter.run(withDebugger: true);
     await breakInBuildMethod(flutter);
     await evaluateComplexReturningExpressions(flutter);
-    await cleanProject();
+  });
+
+    testWithoutContext('evaluating invalid expressions throws an exception with compilation error details', () async {
+    await flutter.run(withDebugger: true);
+    await breakInTopLevelFunction(flutter);
+    await evaluateInvalidExpression(flutter);
   });
```

Comment:
```suggestion
      await flutter.run(withDebugger: true);
      await breakInTopLevelFunction(flutter);
      await evaluateInvalidExpression(flutter);
```

Reply:
I'm still figuring out how to do suggestions from VSCode properly, so this is wrong. I mean to try to fix the indentation here

Reply:
second try:
```suggestion
    await flutter.run(withDebugger: true);
    await breakInTopLevelFunction(flutter);
    await evaluateInvalidExpression(flutter);
```

Reply:
I'll just update it myself... :)